segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperConfig.java

```diff
@@ -48,6 +49,7 @@
     public static final Property<Integer> BK_MIN_NUM_RACKS_PER_WRITE_QUORUM = Property.named("minNumRacksPerWriteQuorum", 2);
     public static final Property<String> BK_NETWORK_TOPOLOGY_SCRIPT_FILE_NAME = Property.named("networkTopologyScriptFileName",
             "/opt/pravega/scripts/sample-bookkeeper-topology.sh");
+    public static final Property<String> BK_DIGEST_TYPE = Property.named("digestType", "CRC32C");
```

Comment:
Instead of using this string, can we use `BookKeeper.DigestType.CRC32.name()`?

Reply:
Done.